packages/cloud_firestore/cloud_firestore/example/lib/main.dart

```diff
@@ -10,7 +10,7 @@ import 'firebase_options.dart';
 
 /// Requires that a Firestore emulator is running locally.
 /// See https://firebase.flutter.dev/docs/firestore/usage#emulator-usage
-bool shouldUseFirestoreEmulator = true;
+bool shouldUseFirestoreEmulator = false;
```

Comment:
What this change intentional?

Reply:
Yes, it shouldn't have been changed in the first place 😅